### PR TITLE
[sdk-vpp#314] Update tests to new VPP forwarder

### PR DESCRIPTION
**Pull request by Bolodya1997**
## Description
Use VPP Forwarder with SR-IOV capabilities instead of SR-IOV Forwarder in all tests.

## Issue link
Needed for networkservicemesh/sdk-vpp#314.

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [x] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: examples/basic/README.md [hunk 1]
```diff
@@ -50,7 +50,27 @@ kubectl exec -n spire spire-server-0 -- \
...
+metadata:
+  name: forwarder-vpp
+spec:
+  template:
+    spec:
+      containers:
+        - name: forwarder-vpp
+          env:
+            - name: NSM_SRIOV_CONFIG_FILE
+              value: /dev/null
```
Comment (edwarnicke) [inline examples/basic/README.md:69]: Question... why does this have to be set... if you don't configure NSM_SRIOV ... shouldn't it just not be used?

Comment (Bolodya1997) [reply examples/basic/README.md]: You are totally right, fixed `cmd-forwarder-vpp` just not to use SR-IOV if there is no config - https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/343.

**Review by denis-tingaikin — APPROVED** [review]
LGTM

@edwarnicke Do you have any additional comments on this?

File: examples/multiforwarder/README.md [hunk 2]
```diff
@@ -1,42 +0,0 @@
-## Requires
```
Comment (denis-tingaikin) [inline examples/multiforwarder/README.md:1]: Can we just disable this example?

Comment (edwarnicke) [reply examples/multiforwarder/README.md:1]: @pperiyasamy How are you on getting the cmd-forwarder-ovs into a place we could put it in the examples?

Comment (edwarnicke) [reply examples/multiforwarder/README.md:1]: @denis-tingaikin One option is to simply use the cmd-forwarder-vpp and not provide it a SRIOV config (and thus it won't handle SRIOV case).

Comment (pperiyasamy) [reply examples/multiforwarder/README.md:1]: @edwarnicke The cmd-forwarder-ovs runs only in [kernel datapath](https://github.com/networkservicemesh/cmd-forwarder-ovs/blob/main/main.go#L227-L231) if no SRIOV config provided. when sriov config exists, then it supports both kernel and smart nic, the connection type would be chosen based on client service request URL.

**Pull request merged**